components/admin_console/group_settings/group_details/group_profile.jsx

```diff
@@ -44,8 +44,7 @@ export default class GroupProfile extends React.PureComponent {
                         </div>
                         <input
                             type='text'
-                            id={customID}
```

Comment:
also this id is being applied at the root of this component and was also being applied to the input -- ids should be unique so I removed it here since it wasnt required